src/material-experimental/mdc-autocomplete/autocomplete.ts

```diff
@@ -31,7 +31,8 @@ import {
   exportAs: 'matAutocomplete',
   inputs: ['disableRipple'],
   host: {
-    'class': 'mat-mdc-autocomplete'
+    'class': 'mat-mdc-autocomplete',
+    '[attr.aria-label]': 'ariaLabel || null',
```

Comment:
This binding can be removed since it doesn't do anything. We take all the content out of the `mat-autocomplete` element and we transfer it to the overlay so this element will be empty.